To: greg@kroah.com
Message-ID: <20060110211547.GM26221@austin.ibm.com>
From: linas@austin.ibm.com (linas)
Cc: linuxppc64-dev@ozlabs.org, linux-pci@atrey.karlin.mff.cuni.cz,
	johnrose@us.ibm.com, Olaf Hering <olh@suse.de>
Subject: [PATCH 1/2] PCI Hotplug/powerpc: module build break


Greg, please apply and forward upstream;
--linas

The RPAPHP hoplug driver will not build as a module, because it calls
on pci_claim_resource(), which is not exported. This exports the symbol.
Problem reported by Olaf Hering <olh@suse.de>

A grep indicates that building drivers/parisc/lba_pci.c
would have trouble building as a module for the same reason.

Signed-off-by: Linas Vepstas <linas@austin.ibm.com>
Signed-off-by: Olaf Hering <olh@suse.de>


Index: linux-2.6.15-git6/drivers/pci/setup-res.c
===================================================================
--- linux-2.6.15-git6.orig/drivers/pci/setup-res.c	2006-01-10 15:02:48.000000000 -0600
+++ linux-2.6.15-git6/drivers/pci/setup-res.c	2006-01-10 15:09:45.547803640 -0600
@@ -112,6 +112,7 @@
 
 	return err;
 }
+EXPORT_SYMBOL_GPL(pci_claim_resource);
 
 int pci_assign_resource(struct pci_dev *dev, int resno)
 {
_______________________________________________
Linuxppc64-dev mailing list
Linuxppc64-dev@ozlabs.org
https://ozlabs.org/mailman/listinfo/linuxppc64-dev

